source/javascripts/controllers/_MainController.js.erb

```diff
@@ -59,6 +61,21 @@
 				viewModel.menuSelected(initialMenu);
 			}
 
+			function checkRepoYmlSplashModalVisibility () {
```

Comment:
I think `checkRepoYmlSplashModalVisibility` indicates it will just do checks, not alter anything, whereas it does.

```suggestion
			function showRepoYmlSplashModalIfNecessary () {
```